### PR TITLE
cleanup(infra.ci): remove AWS arm64 vm replaced by azure ones

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/3471
removing AWS arm64 vm

### CODE REVIEW

File: config/jenkins_infra.ci.jenkins.io.yaml [hunk 1]
```diff
@@ -226,49 +226,6 @@ controller:
                         resourceRequestMemory: "8G"
                     label: "container kubernetes node ruby webbuilder"
                     yamlMergeStrategy: "merge"
-            - amazonEC2:
-                cloudName: "aws-us-east-2"
-                credentialsId: "ec2-agents-credentials"
```
Comment (dduportal) [inline config/jenkins_infra.ci.jenkins.io.yaml:231]: Credential to remove from the top-level list of credential (same file)

File: config/jenkins_infra.ci.jenkins.io.yaml [hunk 2]
```diff
@@ -226,49 +226,6 @@ controller:
...
-                instanceCapStr: "50"
-                noDelayProvisioning: true
-                region: "us-east-2"
-                sshKeysCredentialsId: "ec2-agents-privkey"
```
Comment (dduportal) [inline config/jenkins_infra.ci.jenkins.io.yaml:235]: Credential to remove from the top-level list of credential (same file)

**Review by dduportal — CHANGES_REQUESTED** [review]
Nice work!

Could you also remove the 2 credentials (token used to access AWS API, and SSH key used to access the agents with SSH), unless you find another occurence.


No need to remove the secrets associated to these credentials though: better to wait for this change to be deployed

**Comment by dduportal** [comment]
Nit: I would change `refactor`in the title by `cleanup` as it is not a refactor

**Comment by smerle33** [comment]
> Nit: I would change `refactor`in the title by `cleanup` as it is not a refactor

I inspired my title with https://stackoverflow.com/questions/48075169/semantic-commit-type-when-remove-something

**Review by dduportal — APPROVED** [review]

**Pull request merged**